CHANGELOG.rst

```diff
@@ -1,6 +1,12 @@
 Changelog
 ---------
 
+0.1.6 (2024-04-25)
+******************
+
+- :py:meth:`Optimization.run()<ansys.simai.core.data.optimizations.OptimizationDirectory.run>` will now error out if no workspace is provided and no configured one could be found
```

Comment:
suggestion:  ... now raises an exception if no workspace is provided or no configured workspace is found

Reply:
Hi @tmpbeing, two suggestions (but feel free to ignore if we loose information by being too synthetic):

- The method :py:meth:`Optimization.run()<ansys.simai.core.data.optimizations.OptimizationDirectory.run>` now raises an exception if no configured workspace is provided or found. 

- The method :py:meth:`Optimization.run()<ansys.simai.core.data.optimizations.OptimizationDirectory.run>` now raises an exception if no workspace is provided or none is configured.